pymc4/distributions/discrete.py

```diff
@@ -128,6 +130,75 @@ def upper_limit(self):
         return self.conditions["total_count"]
 
 
+class BetaBinomial(BoundedDiscreteCompoundDistribution):
+    r"""Bounded Discrete compound Beta-Binomial Random Variable
+
+    The pmf of this distribution is
+
+    .. math:: f(x \mid n, \alpha, \beta) = n \choose x \frac{B(x + \alpha, n - x + \beta)}{B(\alpha, \beta)}
```

Comment:
Thanks for copying over the docstring from pymc3, but we have a parameter name discrepancy. Here, we use `concentration` instead of alpha and beta, and `total_count` instead of n. People will be confused by the equation not matching the function's signature. I'm not sure what's best to do here. We can either change the parameter names in the equation or we could add a table after the equation that indicates the correspondence between the equation variables and the parameters in the class' signature.

Reply:
I believe that would make the docstring un-necessarily long but we can just add a docstring saying ``n = total_counts, \alpha = concentration0 and \beta = concentration1``.

Reply:
Ok, lets add that, just make sure that sphinx will render the `total_counts` and `concentration*` as [inline code](https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#inline-markup), and the other stuff as [inline math](http://www.sphinx-doc.org/es/stable/ext/math.html)

Reply:
On it!